Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN set -eux; \
     sudo mkdir -p /usr/lib/jvm/oracle8; \
     curl -L --fail "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246284_165374ff4ea84ef0bbd821706e29b123" | sudo tar -xvzf - -C /usr/lib/jvm/oracle8 --strip-components 1
 
+# GraalVM with native image support
+FROM ghcr.io/graalvm/graalvm-ce:22.2.0 AS graalvm-native-image
```

Comment:
could we use a tag which includes the JDK version and also include that JDK version in the aliased name?

that way we could have native installs of both 11 and 17

Reply:
Yes, I will switch it from `22.2.0` to `ol8-java17-22.2.0` (the `java17-22.2.0` from the official document does not seem to exist on the GH registry).
And I will install the JDK11 variant too.